save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/DeleteOrganizationButton.kt

```diff
@@ -17,25 +17,16 @@ import react.dom.html.ReactHTML.button
 import react.dom.html.ReactHTML.div
 import react.useState
 
-
 /**
  * Button for delete organization
  *
  * @return noting
  */
 val deleteOrganizationButton: FC<DeleteOrganizationButtonProps> = FC { props ->
-    val displayButtonEntry = {}
     val windowOpenness = useWindowOpenness()
-    val (displayTitle, putDisplayTitle) = useState<String?>(null)
-    val (displayMessage, putDisplayMessage) = useState<String?>(null)
-    val (displayButton, putDisplayButtons) = useState(displayButtonEntry)
-
-    val displayButtonErrors = {
-        buttonBuilder("Ok") {
-            windowOpenness.closeWindow()
-        }
-    }
-
+    val (displayTitle, putDisplayTitle) = useState("")
```

Comment:
your putter is usually called setter. just in case

Reply:
Done